src/actions/etherspotActions.js

```diff
@@ -299,3 +296,76 @@ export const unsubscribeToEtherspotNotificationsAction = () => {
     etherspotService.unsubscribe();
   };
 };
+
+const handleAccountUpdatedNotification = (
+  notification: any,
+  chain: Chain,
+  dispatch: Dispatch,
+) => {
+  dispatch(refreshEtherspotAccountsAction());
+};
+
+const handleGatewayBatchUpdatedNotification = async (
```

Comment:
Extracted from `subscribeToEtherspotNotificationsAction` callback - did not modify the logic.